packages/landing/src/components/map.switcher.tsx

```diff
@@ -54,12 +54,12 @@ export class MapSwitcher extends Component {
 
   getStyleType(): { layerId: string; style?: string } {
     if (Config.map.layerId === 'aerial') {
-      return { layerId: 'topographic', style: 'topographic' };
-    } else if (Config.map.layerId === 'topographic' && Config.map.style === 'topographic') {
+      return { layerId: 'topographic', style: 'topographic-v2' };
```

Comment:
I believe LayerId should be `topograhpic-v2` as well, if the combination of v1 and v2 works there might be some bugs in the system not fetch the correct layerId if style exists.

Reply:
I believe the layer ID should point to the tilesetId and Style should point to the styleID, not sure why these combination works though.

Reply:
Happy to merge the changes. Did you manged to find out why the combination of `i=topographic` and `s=topograhpic-v2` work? Otherwise, we could create a bug ticket here too.

Reply:
I think that's some legacy debt there, we should move everything to stlyeId only.